tests/unittests/reporting/test_reporting.py

```diff
@@ -164,56 +164,49 @@ def test_has_result_with_optional_post_files(self):
             name, desc, result, post_files=files
         )
         ret = event.as_dict()
-        self.assertTrue("result" in ret)
-        self.assertTrue("files" in ret)
-        self.assertEqual(ret["result"], result)
+        assert "result" in ret
+        assert "files" in ret
+        assert ret["result"] == result
         posted_install_log = ret["files"][0]
-        self.assertTrue("path" in posted_install_log)
-        self.assertTrue("content" in posted_install_log)
-        self.assertTrue("encoding" in posted_install_log)
-        self.assertEqual(posted_install_log["path"], files[0])
-        self.assertEqual(posted_install_log["encoding"], "base64")
+        assert "path" in posted_install_log
+        assert "content" in posted_install_log
+        assert "encoding" in posted_install_log
+        assert posted_install_log["path"] == files[0]
+        assert posted_install_log["encoding"] == "base64"
 
 
-class TestBaseReportingHandler(TestCase):
+class TestBaseReportingHandler:
     def test_base_reporting_handler_is_abstract(self):
-        regexp = r".*abstract.*publish_event.*"
-        self.assertRaisesRegex(TypeError, regexp, handlers.ReportingHandler)
+        with pytest.raises(TypeError, match=r".*abstract.*publish_event.*"):
```

Comment:
This test tests that ReportingHandler is an abstract class, mypy complains about instantiating and abstract class.

Honestly, I think this test doesn't add much value and I suggest removing it.